### PR TITLE
Aspect Fit in imageset item view

### PR DESCRIPTION
**Pull request by urmitchauhan**
- change card size
- add item in testutils
- fix Aspect fit issue

**Before**    |  **After**
:-------------------------:|:-------------------------:
Screenshot 2022-11-29 at 8 26 18 PM | Screenshot 2022-12-01 at 7 14 42 PM
 Screenshot 2022-11-29 at 8 26 36 PM | Screenshot 2022-12-01 at 7 15 03 PM

## Related Issue
Please use one of the well-known [github fixes keywords](https://help.github.com/en/articles/closing-issues-using-keywords) to reference
the issues fixed with this PR (eg Fixes #<github issue number>). If an issue doesn't yet exist please create one if reasonable to aid 
in issue tracking.

**NOTE**: For multiple issues resolved by this PR use the corresponding keywords **every time** in a comma-delimited list per the reference
page above.

## Description
Create a [descriptive title and provide all of the relevant details in the description](https://chris.beams.io/posts/git-commit/). 
This information lives with the code and will help future readers (***including yourself***) and will serve as documentation.

At the very least please describe the issue you are addressing, and what your fix entails. 

## Sample Card
If appropriate, please include a link to a card in one of the samples directories that can be used to validate this change. This can be an existing card or a card added with this PR.

## How Verified
How you verified the fix, including one or all of the following: 
1. New unit tests that were added if any. If none were added please add a quick line explaining why not.
2. Existing relevant unit/regression tests that you ran
3. Manual scenario verification if any; ***Do include .gif's or screenshots of the testing you performed here if you think that it 
will aid in code reviews or corresponding fixes on other platforms for eg.***

### CODE REVIEW

File: source/macos/ADCMacOSVisualizer/ADCMacOSVisualizer/AppDelegate.swift [hunk 1]
```diff
@@ -4,7 +4,7 @@ class AppDelegate: NSObject, NSApplicationDelegate {
     var window: NSWindow?
 
     func applicationDidFinishLaunching(_ aNotification: Notification) {
-        let minWindowSize = NSSize(width: 840, height: 540)
+        let minWindowSize = NSSize(width: 1366, height: 768)
```
Comment (mukuagar) [inline source/macos/ADCMacOSVisualizer/ADCMacOSVisualizer/AppDelegate.swift:7]: Why did we change this?

File: source/macos/ADCMacOSVisualizer/ADCMacOSVisualizer/RootViewController.xib [hunk 2]
```diff
@@ -131,7 +131,7 @@
                     </scroller>
                 </scrollView>
                 <comboBox verticalHuggingPriority="750" translatesAutoresizingMaskIntoConstraints="NO" id="O4M-uk-PGq">
-                    <rect key="frame" x="853" y="4" width="174" height="27"/>
+                    <rect key="frame" x="873" y="7" width="154" height="23"/>
```
Comment (mukuagar) [inline source/macos/ADCMacOSVisualizer/ADCMacOSVisualizer/RootViewController.xib:134]: Does this file need changing?

File: source/macos/ADCMacOSVisualizer/ADCMacOSVisualizer/AppDelegate.swift (hunk 1 above)
Comment (urmitchauhan) [reply source/macos/ADCMacOSVisualizer/ADCMacOSVisualizer/AppDelegate.swift:7]: Screen size is looking too small, so.

File: source/macos/ADCMacOSVisualizer/ADCMacOSVisualizer/RootViewController.xib (hunk 2 above)
Comment (urmitchauhan) [reply source/macos/ADCMacOSVisualizer/ADCMacOSVisualizer/RootViewController.xib:134]: yes I've changed card size - 350 to 432.

**Review by mukuagar — APPROVED** [review]

**Pull request merged**